### PR TITLE
Pt signoff reports

### PR DESCRIPTION
**Pull request by alexcarsello**
Small quality of life improvements for pt-timing-signoff node.

-Create separate setup and hold reports, rather than one combined report
-Include full clock paths in both reports
-Include transition times in both reports
-Organizes all reports into reports folder

Also adds pt-timing signoff to gcd flow.

### CODE REVIEW

File: steps/synopsys-pt-timing-signoff/pt.tcl [hunk 1]
```diff
@@ -9,22 +9,23 @@ set report_default_significant_digits 3
 ##################################################################
 #    Constraint Analysis Section
 ##################################################################
```
Comment (ctorng) [inline steps/synopsys-pt-timing-signoff/pt.tcl:11]: Can you add your name to the top of all the files in this node? Look at the other nodes where there is a header.

Can you also use the same coding convention?

#------------------------------------------------------------------------
# Constraint Analysis Section
#-----------------------------------------------------------------------

File: steps/synopsys-pt-timing-signoff/pt.tcl [hunk 2]
```diff
@@ -9,22 +9,23 @@ set report_default_significant_digits 3
...
+check_timing -verbose > reports/check_timing.report
 
 ##################################################################
 #    Report_timing Section                                       #
 ##################################################################
-report_global_timing > report_global_timing.report
-report_clock -skew -attribute > report_clock.report 
-report_analysis_coverage > report_analysis_coverage.report
-report_timing -crosstalk_delta -slack_lesser_than 1000.0 -max_paths 100 -pba_mode exhaustive -delay min_max -nosplit -input -net > report_timing_pba.report
+report_global_timing > reports/report_global_timing.report
```
Comment (ctorng) [inline steps/synopsys-pt-timing-signoff/pt.tcl:24]: And add a new line here. Let's just make all the standard nodes coding convention similar.

**Review by ctorng — APPROVED** [review]

File: steps/synopsys-pt-timing-signoff/pt.tcl [hunk 3]
```diff
@@ -9,22 +9,23 @@ set report_default_significant_digits 3
...
+report_clock -skew -attribute > reports/report_clock.report 
+report_analysis_coverage > reports/report_analysis_coverage.report
+report_timing -crosstalk_delta -slack_lesser_than 1000.0 -max_paths 100 -pba_mode exhaustive -delay max -nosplit -input -net -transition_time -path_type full_clock_expanded > reports/report_timing_setup_pba.report
+report_timing -crosstalk_delta -slack_lesser_than 1000.0 -max_paths 100 -pba_mode exhaustive -delay min -nosplit -input -net -transition_time -path_type full_clock_expanded > reports/report_timing_hold_pba.report
```
Comment (ctorng) [inline steps/synopsys-pt-timing-signoff/pt.tcl:29]: Can we also name the reports similarly to the DC node?

- like .timing.setup.rpt
- and .timing.hold.rpt

File: steps/synopsys-pt-timing-signoff/pt.tcl (hunk 1 above)
Comment (alexcarsello) [reply steps/synopsys-pt-timing-signoff/pt.tcl:11]: This node was already violating the coding convention used in the rest of the nodes, but yes I can bring it in line with the others.

**Pull request merged**